src/os/my_cpuid_wine.c

```diff
@@ -1,15 +1,548 @@
 #include <windows.h>
 
 #include "my_cpuid.h"
+#include "../emu/x64emu_private.h"
+#include "debug.h"
+#include "freq.h"
+
+NTSYSAPI ULONG WINAPI NtGetTickCount(VOID);
+NTSYSAPI ULONG NTAPI RtlRandom(ULONG *seed);
+
+static int nCPU = 0;
+
+void grabNCpu() {
+    SYSTEM_INFO sysinfo;
+    GetSystemInfo(&sysinfo);
+    nCPU = sysinfo.dwNumberOfProcessors;
+}
+int getNCpu()
+{
+    if(!nCPU)
+        grabNCpu();
+    if(BOX64ENV(maxcpu) && nCPU>BOX64ENV(maxcpu))
+        return BOX64ENV(maxcpu);
+    return nCPU;
+}
 
 const char* getBoxCpuName()
 {
-    return NULL;
+    static char branding[] = "libwowbox64.dll";
+    return branding;
 }
 
 void my_cpuid(x64emu_t* emu, uint32_t tmp32u)
```

Comment:
why duplicating this code?

Reply:
It's exactly the same code as the linux version right?

Reply:
true, would you rather have a my_cpuid_common.c for that function or only one file with ifdefs?

Reply:
Which suggestion? : D